core/jazz_ui/e2e/04-apiservices/apiservices.e2e-spec.ts

```diff
@@ -0,0 +1,316 @@
+
+const twok = 2000;
+const fivek = 5000;
+const tenk = 10000;
+const fifteenk = 15000;
+const twentyk = 20000;
+const thirtyk = 30000;
+const sixtyk = 60000;
+
+const timeOutHigh = 180000;
+const emailId = CONFIGURATIONS.optional.general.e2e.EMAIL_ID;
+const config = CONFIGURATIONS.optional.general.e2e;
+
+describe('Overview', () => {
+  let jazzServices_po: Jazz;
+
+  const EC = protractor.ExpectedConditions;
+  let winhandle;
+  let servicename;
+  let test;
+
+  beforeAll(() => {
+    jazzServices_po = new Jazz();
+
+  });
+
+  function createservice(servicename) {
+    jazzServices_po.getServiceName().sendKeys(servicename);
+    jazzServices_po.getNameSpace().sendKeys('jazztest');
+    jazzServices_po.getServiceDescription().sendKeys('Testing');
+  }
+
+  function serviceapprover() {
+    browser.driver.sleep(fivek);
+    jazzServices_po.getSubmit().click();
+    fluentwaittry(jazzServices_po.getDone(), tenk);
+    jazzServices_po.getDone().click();
+  }
+  function waitForSpinnerDisappear() {
+    browser.wait(EC.not(EC.visibilityOf(jazzServices_po.getSpinner())), 180000);
+  }
+  function waitForMetricsSpinner() {
+    browser.wait(EC.not(EC.visibilityOf(jazzServices_po.getMetricsSpinner())), 180000);
+  }
+  function fluentwaittry(ele, t) {
+    browser.manage().timeouts().implicitlyWait(0);
+    browser.wait(function () {
+      browser.sleep(t);
+      return ele.isDisplayed()
+        .then(
+          function (text) {
+            return text;
+
+          },
+          function (error) {
+            browser.refresh();
+            return error;
+          });
+    }, 180 * 1000);
+  }
+
+  function waitforservice(ele, t) {
+    browser.manage().timeouts().implicitlyWait(0);
+    browser.wait(function () {
+      browser.sleep(t);
+      return ele.isDisplayed()
+        .then(
+          function (text) {
+            return text;
+
+          },
+          function (error) {
+            browser.refresh();
+            return false;
+          });
+    }, 240 * 1000);
+  }
+
+  function refreshbutton(ele, t) {
+    browser.manage().timeouts().implicitlyWait(0);
+//     if (browser.wait(EC.elementToBeClickable(jazzServices_po.getRefresh()), fivek))
+//     {
+//       return false;
+//     }
+    browser.wait(function () {
+      browser.sleep(t);
+      return ele.isDisplayed()
+        .then(
+          function (text) {
+            return text;
+          },
+          function (error) {
+            jazzServices_po.getRefresh().click();
+            return false;
+          });
+    }, 120 * 1000);
+  }
+
+
+  it('Create API Service', function () {
+    browser.driver.sleep(fivek);
+    browser.wait(EC.visibilityOf(jazzServices_po.getCreateService()), timeOutHigh);
+    browser.wait(EC.elementToBeClickable(jazzServices_po.getCreateService()), timeOutHigh);
+    winhandle = browser.getWindowHandle();
+    //To create Service-API
+    jazzServices_po.getCreateService().click();
+    var min = 111111111;
```

Comment:
shift to a configuration file

Reply:
kept in that files and tried it was not working